website/docs/components/reveal/partials/code/component-api.md

```diff
@@ -0,0 +1,16 @@
+## Component API
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="text" @required="true" @type="string">
+    Plain text string which will appear on the toggle button which controls hiding and showing of the content. If no text value is defined an error will be thrown.
+  </C.Property>
+  <C.Property @name="textWhenOpen" @type="string">
+    Plain text which displays on the toggle button while the content is displayed.
+  </C.Property>
+  <C.Property @name="isOpen" @values={{array "false" "true" }} @default="false" @type="boolean">
```

Comment:
I think [we now consider boolean values redundant](https://github.com/hashicorp/design-system/pull/1369/files), so only having the default should do.